source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
Little abstraction over `window`, because when we import package on server, `set` throws error, because `window` undefined. Possible variants: `if` check, invoke `set` after `initiate`.